app/status/views.py

```diff
@@ -11,14 +12,16 @@ def status_no_db():
     try:
         return jsonify(
             status="ok",
-            version=utils.get_version_label(),
-            db_version=utils.get_db_version()
+            version=get_version_label(),
+            db_version=utils.get_db_version(),
```

Comment:
If we're removing the `utils.` from other lines then we could also add `get_db_version to the imports and drop it for this line?